ml-service/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.9.7
+
+COPY train.py /app/
+RUN pip install numpy scikit-learn
```

Comment:
please move installation before 

```
COPY train.py /app/
```
reasons:  invalidating the Docker build cache
see full explanation: https://stackoverflow.com/questions/34398632/docker-how-to-run-pip-requirements-txt-only-if-there-was-a-change